### DISCUSSION TITLE
Stylesheet not being used?

### DISCUSSION THREAD

**Discussion by Hoogin** [firefly-iii/firefly-iii#5466] [Firefly III] [answered]
Happy new year!
I've installed firefly and when I login the new-user screen looks like this.

![Screenshot_20220102_113723](https://user-images.githubusercontent.com/12593971/147864665-6a0c3f65-2571-4c2a-ac9b-aa4b24dd5cd3.png)

Any ideas what could be wrong? Permissions? Apache settings?

**Comment by JC5** [comment, marked as answer]
https://github.com/firefly-iii/firefly-iii/issues/5464

**Reply by Hoogin** [reply]
Thank you. I'll wait for that new version.
